### PR TITLE
Issue #2201 Fix CentOS CI job is failing in building docs

### LINKED ISSUE #2201: CentOS CI job is failing in building docs for master branch
```
15:35:23 Building OpenShift Origin for branch 'master'
15:35:23   - welcome/index
15:35:23 rake aborted!
15:35:23 NameError: undefined local variable or method `repo_path' for #<AsciiBinder::TemplateRenderer:0x000000024c5ca0>
15:35:23 /home/minishift_ci/gopath/src/github.com/minishift/minishift/openshift-docs/_templates/page.html.erb:25:in `block in singleton class'
15:35:23 /home/minishift_ci/gopath/src/github.com/minishift/minishift/openshift-docs/_templates/page.html.erb:-31:in `instance_eval'
15:35:23 /home/minishift_ci/gopath/src/github.com/minishift/minishift/openshift-docs/_templates/page.html.erb:-31:in `singleton class'
```

Ref: https://ci.centos.org/job/minishift/535/console

### PR DESCRIPTION
**Pull request by budhrg**
Fix #2201

### CODE REVIEW

**Comment by budhrg** [comment]
@LalatenduMohanty Fix working now. https://ci.centos.org/job/minishift-pr/3943/consoleFull

File: centos_ci.sh [hunk 1]
```diff
@@ -155,7 +155,7 @@ function install_docs_prerequisite_packages() {
   rvm install ruby-2.2.5
   echo "CICO: RVM and Ruby Installed"
 
-  gem install ascii_binder -v 0.1.9
+  gem install ascii_binder -v 0.1.13
```
Comment (praveenkumar) [inline centos_ci.sh]: @budhrg what issue new version of ascii_binder resolve for doc building, can you add it to commit message?

**Comment by LalatenduMohanty** [comment]
@budhrg How to cross-check this will fix the master build issue?

**Comment by budhrg** [comment]
> How to cross-check this will fix the master build issue?

@LalatenduMohanty Created https://github.com/minishift/minishift/pull/2270 to test in CI. Otherwise, only way to test is by merging this PR.

Once #2270 is passing and successfully building docs, we can close that one and merge this PR.

**Pull request merged**